### PR TITLE
Make the distribution overview strings can be translated

### PR DESCRIPTION
**Pull request by monstorix**
The strings in the overview tabs of per-distribution pages are currently hardcoded. This PR will move out the strings into locale files for better localization.

This PR will also remove duplicated entries for Simplified and Traditional Chinese. It may confuse the user for having multiple links in the same language.

## Screenshots for Chinese Entries
|Before|After|
|-----|-----|
|Screenshot-bf|Screenshot-aft|

### CODE REVIEW

File: _data/locales/en.yml [hunk 1]
```diff
@@ -395,4 +395,17 @@ Tumbleweed_features:
   gamepad:
     name: Gaming
     description: |
-     Tumbleweed provides users with the latest gaming-related software. With the Linux support for gaming improving rapidly, that's an important aspect for smooth, performant and problem free experience in your favorite games.
\ No newline at end of file
+     Tumbleweed provides users with the latest gaming-related software. With the Linux support for gaming improving rapidly, that's an important aspect for smooth, performant and problem free experience in your favorite games.
+Leap_ovw_title: A brand new way of building openSUSE and a new type of a hybrid Linux distribution
```
Comment (hellcp) [inline _data/locales/en.yml:399]: We tend to avoid capitalizing strings here

File: _data/locales/en.yml [hunk 2]
```diff
@@ -395,4 +395,17 @@ Tumbleweed_features:
...
+Leap_ovw_desc: Leap uses source from SUSE Linux Enterprise (SLE), which gives Leap a level of stability unmatched by other Linux distributions, and combines that with community developments to give users, developers and sysadmins the best stable Linux experience available.
+Tumbleweed_ovw_title: State-of-the-art desktop and server operating system
+Tumbleweed_ovw_desc: With Tumbleweed you don't have to take difficult decisions about things you value, either freedom or safety, either control or security, technology or stability -- Tumbleweed lets you have your cake and eat it too!
+LeapMicro_ovw_title: Leap Micro is an ultra-reliable, lightweight operating system built for containerized and virtualized workloads.
```
Comment (hellcp) [inline _data/locales/en.yml:403]: Let's make it `leap_micro`

File: _data/locales/en.yml (hunk 1 above)
Comment (monstorix) [reply _data/locales/en.yml]: done

File: _data/locales/en.yml (hunk 2 above)
Comment (monstorix) [reply _data/locales/en.yml]: done

**Pull request merged**